plugins/deepgram/package.json

```
@@ -20,8 +20,10 @@
     "typescript": "^5.0.0"
   },
   "dependencies": {
-    "@livekit/agents": "workspace:*",
-    "@livekit/rtc-node": "^0.11.1",
     "ws": "^8.16.0"
+  },
+  "peerDependencies": {
+    "@livekit/agents": "workspace:*",
```

```suggestion
    "@livekit/agents": "workspace:^",
```

so that it doesn't request the exact version and error if not found
this will match 0.x.y to 0.x+1.0

---

good point, but should we instead maybe use `~` so that it doesn't require the one set in the workspace as a minimum?

---

it would still require it as a minimum — in 0.x they are equivalent, ~ does it until next minor and ^ until next major
if we want to just accept any value shipped with agents (which should be fine unless you're manually upgrading only one for some reason), we can do simply `*` without `workspace:`

---

oh, you're right, I always had a understanding of _any_ patch version for tilde.